src/actions/userManagement.js

```diff
@@ -7,9 +7,10 @@ import {
   USER_PROFILE_DELETE,
 } from '../constants/userManagement';
 import { ENDPOINTS } from '../utils/URL';
-import { UserStatus } from '../utils/enums';
+import { UserFinalDayStatus, UserStatus } from '../utils/enums';
```

Comment:
`UserFinalDayStatus` is not existing, pls remove it.

Reply:
thanks, i removed it